### PR TITLE
Renew button fixes

### PR DESCRIPTION
**Pull request by SebaSOFT**
- Update SDK with support for new functionality
- Fixed a problem with subscription renewal buttons not beeing shown on certain scenarios

### CODE REVIEW

File: plenigo_plugin/PlenigoShortcodeManager.php [hunk 1]
```diff
@@ -102,14 +102,31 @@ public function plenigo_handle_shortcode($atts, $content = null, $tag = null) {
...
-            ), $atts);
+                ), $atts);
 
         $btnTitle = $a['title'];
         $cssClass = $a['class'];
         $prodId = $a['prod_id'];
-        if ($tag !== 'pl_checkout_button' && $prodId !== "" && PlenigoSDKManager::get()->plenigo_bought($prodId)) { //Return the content untouched
-            return do_shortcode($content);
-        } else { //Do the plenigo checkout button
+        $isIgnoringTag = ($tag == 'pl_checkout_button' || $tag == 'pl_renew');
```
Comment (robertholfeld) [inline plenigo_plugin/PlenigoShortcodeManager.php]: what is with the $prodId !== "" check?

File: plenigo_plugin/PlenigoShortcodeManager.php [hunk 2]
```diff
@@ -102,14 +102,31 @@ public function plenigo_handle_shortcode($atts, $content = null, $tag = null) {
...
+
+        //evaluate the condition
+        $renderButton = true;
+        if ($isIgnoringTag) {
+            plenigo_log_message("Got ignoring chosrtcode, rendering button", E_USER_NOTICE);
```
Comment (robertholfeld) [inline plenigo_plugin/PlenigoShortcodeManager.php]: chosrtcode?

File: plenigo_plugin/PlenigoShortcodeManager.php [hunk 3]
```diff
@@ -102,14 +102,31 @@ public function plenigo_handle_shortcode($atts, $content = null, $tag = null) {
...
+            $renderButton = true;
+        } else {
+            plenigo_log_message("Shosrtcode not ignoring, checking if bought", E_USER_NOTICE);
```
Comment (robertholfeld) [inline plenigo_plugin/PlenigoShortcodeManager.php]: Shosrtcode?

Comment (SebaSOFT) [inline plenigo_plugin/PlenigoShortcodeManager.php]: I I will take my boxing gloves off when typing...

**Pull request merged**